content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,30 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Documentation content standards
+
+Kubernetes documentation does not host certain types of content:
+
+* Vendor, provider, or project-specific content from projects that are not:
+
+    * Inside the kubernetes or kubernetes-sigs GitHub organizations
+    * Inside the CNCF
+
+* Identical content hosted in multiple locations ("dual-sourced" content)
+
+For example:
+
+| Allowed |                           Action                          |                                                    Reason                                                   |
+|:-------:|:----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+|    Y    | Linking to Kubernetes in Docker (KinD) docs from k8s docs | The KinD project resides in github.com/kubernetes-sigs.                                                     |
+|    N    | Hosting KinD docs in k8s website                          | Dual-sourced content requires duplicated effort from maintainers and tends to go stale/rotten more quickly. |
+|    Y    | Linking to Prometheus docs from k8s docs                  | Prometheus is a CNCF project.                                                                                |
+|    N    | Linking to rkt docs from k8s docs                         | rkt is an (almost) archived CNCF project.                                                                    |
```

Comment:
We can leave out "(almost)" here, to avoid [statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

Reply:
@zacharysarah  how do writers and developers know rkt is going to be archived? I checked rkt's [roadmap](https://github.com/rkt/rkt/blob/master/ROADMAP.md) but I didn't see anyting about an archive date, and rkt is listed an an incubating project on the CNCF project [page](https://www.cncf.io/projects/). Does the CNCF maintain a list of soon-to-be-archived projects? I'd like to add verbiage on where to check for soon-to-be-archived CNCF projects.

Reply:
done